### ISSUE TITLE
Dual PlayAxis on the same page

### ISSUE THREAD

**Issue by DenisB123** [mprozil/PlayAxis#10]
It will me nice to have a dual independent axis rolling each at different speed.
Actually I try to make a visual with rolling team result with the same page of a comment about team player without having both related.
I try different setting with two of your PlayAxis on the same page, but look like one are blooking the other even if I edit interaction for not having interaction between any of the slider / axis / data

**Comment by andresmorago** [comment]
This would be great for a pagination strategy on a table. That way i could for example rotate between states and page numbers as a lower level

**Comment by iTouch61** [comment]
Yes, that would be a great feature. I’m facing the same issue right now and would really like to have this functionality as well.
